cmd/webhook/main.go

```diff
@@ -25,20 +25,20 @@ import (
 
 	"go.uber.org/zap"
 
+	autoscalingv1alpha1 "github.com/knative/serving/pkg/apis/autoscaling/v1alpha1"
+	apiconfig "github.com/knative/serving/pkg/apis/config"
+	net "github.com/knative/serving/pkg/apis/networking/v1alpha1"
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"github.com/knative/serving/pkg/apis/serving/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
```

Comment:
while you're at it, can you split the imports into k8s, knative.dev github.com/knative others groups? :-)